### DISCUSSION TITLE
help with using: instruct mode now allows any number of newlines in the start and end tag, configurable by user.

### DISCUSSION THREAD

**Discussion by lilblam** [LostRuins/koboldcpp#276] [Q&A]
1.33 added this flexibility. So, if the prompt style is:

```
USER: prompt
ASSISTANT:
```

In the Kobold UI, I can add:

```
USER: 
\nASSISTANT: 
```

for Star Seq and End Seq respectively.

Just wanted to ask if this is how it was intended to be used? I can't find any other ways to add new lines or spaces after the start/end sequence other than include all that right in the UI. Can someone confirm?

I only ask cuz that way seems a little "messier" than having some checkbox for newlines etc.

It would be cool if we could set up prompt style and exact syntax right in the .bat file when calling koboldcpp.exe tho!

**Comment by LostRuins** [comment]
Yes that's exactly how to use it. So you can do

```
\n\nUSER:\n\n
\n\nASSISTANT:\n\n
```

if you want the old style similar to "enable newlines" 
and you can customize how many newlines you want.

**Reply by lilblam** [reply]
Awesome that's what I figured, thanks! I think it gives complete control, and unfortunately models are often pretty sensitive to that, so it's a necessary evil.
